src/components/Review.jsx

```diff
@@ -49,6 +61,56 @@ const Review = () => {
     });
   };
 
+  const handleCommentSubmit = event => {
+    event.preventDefault();
+    const commentBody = event.target['0'].value;
+    postComment(user, review.review_id, commentBody)
+      .then(newComment => {
+        setComments(currComments => {
+          const newComments = [...currComments];
+          newComments.unshift(newComment);
+          return newComments;
+        });
+        setIsSubmitted(true);
+      })
+      .catch(err => {
+        if (err.response.data.msg === 'User does not exist') {
+          setUserExists(false);
+        }
+      });
+  };
+
+  let commentContainer;
+
+  isSubmitted
+    ? (commentContainer = <p>comment added!</p>)
+    : (commentContainer = (
+        <div>
+          <h3>Add new comment</h3>
+          <form onSubmit={handleCommentSubmit} id="new-comment">
+            <textarea
+              form="new-comment"
+              placeholder="new comment..."
+              required
+            />
+            <button type="submit">Submit</button>
+          </form>
+        </div>
+      ));
+
+  const handleDelete = event => {
+    event.preventDefault();
+    const commentId = event.target.parentElement.id;
```

Comment:
On deleting we also want to be shown visual indication if delete request is in progress (loading state)

We also would like to see a failure message if unable to delete comment